platform/graphics/platform_graphics.c

```diff
@@ -136,21 +137,20 @@ void plDeleteFrameBuffer(PLFrameBuffer *buffer) {
         return;
     }
 
-#if 0
-    glDeleteFramebuffers(1, &buffer->fbo);
-    glDeleteRenderbuffers(1, &buffer->rbo);
-#endif
+    CallGfxFunction(DeleteFrameBuffer, buffer);
+
+    //TODO: Dealloc object here
 }
 
-void plBindFrameBuffer(PLFrameBuffer *buffer) {
-    if(buffer == NULL) {
-        // todo, warning regarding invalid buffer blah blah
-        return;
-    }
+void plBindFrameBuffer(PLFrameBuffer *buffer, PLFBOTarget targetBinding) {
+    //NOTE: NULL is valid for *buffer, to bind the SDL window default backbuffer
+    CallGfxFunction(BindFrameBuffer, buffer, targetBinding)
+}
 
-#if defined(PL_MODE_OPENGL)
-    glBindFramebuffer(GL_DRAW_FRAMEBUFFER, buffer->fbo);
-#endif
+void plBlitFrameBuffers(PLFrameBuffer *srcBuffer, unsigned int srcW, unsigned int srcH, PLFrameBuffer *dstBuffer, unsigned int dstW, unsigned int dstH, bool linear ) {
```

Comment:
For public API these should be consistent, so use https://en.wikipedia.org/wiki/Snake_case for any public variables.